application/templates/pages/about/performance.md

```diff
@@ -1,20 +1,20 @@
 Performance indicators show how the planning data platform is making it easier to find, use and trust planning and housing data in England.
 
-Last updated 31 July 2024.
+Last updated 28 August 2024.
 
 ## Availability
 
 We index planning and housing datasets provided by multiple organisations in England.
 
 Many datasets have one owner providing the data nationally. There are 8 datasets compiled from data sources provided by local planning authorities (LPAs) across England.
 
-### 89 datasets on the platform
+### 93 datasets on the platform
 
-89 datasets available from 452 organisations. <!--? added last month (+/-0%). -->
+93 datasets available from 377 organisations. <!--? added last month (+/-0%). -->
 
-131 data sources out of 584 provided by 31 local planning authorities. 4 up from last month (+0.7%).
+138 data sources out of 584 provided by 33 local planning authorities. 7 up from last month (+1.2%).
 
-1 new local planning authority providing data. 1 up from last month (+1%).
+2 new local planning authorities providing data this month (+2.7%).
```

Comment:
This is 1 up from last month too, I think.

Reply:
In the page you posted it was 31 I thought? https://www.staging.digital-land.info/about/performance

Reply:
Yes, sorry, this is where I'm trying to introduce a rate again, but perhaps it's not worthwhile.